v22.1/map-sql-activity-to-app.md

```diff
@@ -0,0 +1,57 @@
+---
+title: Map SQL Activity using an Application Name
+summary: How to set an application name and filter SQL activity associated with the application.
+toc: true
+docs_area: develop
+---
+
+Many relational database management systems enable you to programmatically create and tag your SQL connections with a custom name or label. This practice lets you to quickly identify which part of your application is potentially degraded due to database health and design. By identifying problematic transactions and statements, you can quickly trace back to parts of your applications that generated those transactions and statements for a holistic understanding of the application performance impact as well as potential mitigations and optimization opportunities.
+
+## Set the application name
+
+It is best practice to set the application name with CockroachDB. You can do this in the [connection string](connection-parameters.html#additional-connection-parameters) `postgres://root@<servername>:26257/mydb?movr_app` or at the [session level](set-vars.html):
+
+{% include copy-clipboard.html %}
+~~~sql
+SET application_name = movr_app;
+~~~
+
+## Filter by application name
+
+Once you set the application name, the [DB Console](ui-overview.html) lets you [filter database workloads by application name](ui-statements-page.html#filter).
+
+<img src="{{ 'images/v22.1/movr-app.png' | relative_url }}" alt="Movr app filter" style="border:1px solid #eee;max-width:80%" />
+
+If parts of your applications or known microservices are experiencing performance degradation, you can filter for the database workload tracing back to that part of your application directly in the DB Console. You can quickly identify whether there were database performance problems and if so, troubleshoot the issue using native [SQL observability touch points](#trace-sql-activity-using-metrics) in the DB Console.
```

Comment:
Nit: I'd suggest calling out that you filter for statements and transactions in the DB console. I seemed to have missed that part in the google doc we originally had. 

If parts of your applications or known microservices are experiencing performance degradation, you can filter for the database workload tracing statements and transactions back to that part of your application directly in the DB Console. You can quickly identify whether there were database performance problems and if so, troubleshoot the issue using native [SQL observability touch points](#trace-sql-activity-using-metrics) in the DB Console.